pkg/repositories/models/workflow.go

```diff
@@ -15,5 +15,10 @@ type Workflow struct {
 	TypedInterface          []byte
 	RemoteClosureIdentifier string `gorm:"not null" valid:"length(0|255)"`
 	// Hash of the compiled workflow closure
-	Digest []byte
+	Digest        []byte
+	DescriptionID uint `gorm:"index"`
+	// ShortDescription is saved in the description entity table. set this to read only so we won't create this column.
+	// Adding ShortDescription because we want to unmarshal the short description in the
+	// descriptionEntity table to workflow object.
+	ShortDescription string `gorm:"<-:false"`
```

Comment:
Because this is stored in the descriptions table, we'll have to do a left join now on every GetTask and GetWorkflow to populate the short description? Any reason not to store it inline?

Reply:
updated, store it inline now